irods/login/native.py

```diff
@@ -0,0 +1,9 @@
+
+def login(conn):
+    conn._login_native()
+
+# TODO : probably not needed, but interesting.
+from . import X as X_base
+class X(X_base):
+        pass
```

Comment:
Please remove this if it's not doing anything for us.

Reply:
Because of #499, I think I'd like to leave it here.  As a reminder of & guide to  how the client side state machine class will look.

Reply:
Okay. Just in case, please leave a more detailed comment about what it is meant to represent, then (preferably with an issue number, too).

Reply:
Done.